CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Braintree Android SDK Release Notes
 
+## 4.16.0
+
+* PayPalNativeCheckoutClient
+  * Adding in Native checkout support for one time password
```

Comment:
Can we list the version of PayPalCheckout here? Similar to how we track CardinalMobile version updates ([see example](https://github.com/braintree/braintree_android/blob/6e003485313db9eafe32fb55966ddeed444da46f/CHANGELOG.md?plain=1#L55)).

Reply:
yup, done!